crates/typed-store/src/rocks/mod.rs

```diff
@@ -318,17 +377,33 @@ where
 
     #[instrument(level = "trace", skip_all, err)]
     fn insert(&self, key: &K, value: &V) -> Result<(), TypedStoreError> {
+        let _timer = self
```

Comment:
We don't need the timer, but you can name the span better.  IE instead of adding a latency measurement, change the instrument above to say: 

    #[instrument(level = "trace", name = "rocksdb_insert", .... )]

Reply:
same reasoning